otelcollector/deploy/chart/prometheus-collector/templates/prometheus-collector-secretProviderClass.yaml

```diff
@@ -39,5 +41,5 @@ spec:
           objectFormat: pfx                                  # [REQUIRED] DO NOT CHANGE FROM 'pfx'
           objectVersion: ""                                  # [OPTIONAL] object versions, default to latest if empty
       {{- end}}
-    tenantId: {{ required "azureKeyVault.tenantId is required" .Values.azureKeyVault.tenantId | toString | quote }}       # [CHANGE AS APPROPRIATE][REQUIRED] the tenant ID of the KeyVault specified above
+    tenantID: {{ required "azureKeyVault.tenantId is required" .Values.azureKeyVault.tenantId | toString | quote }}       # [CHANGE AS APPROPRIATE][REQUIRED] the tenant ID of the KeyVault specified above
```

Comment:
is it intentional to change `tenantId` to `tenantID` ? I see its being referenced when used as `tenantId` .

Reply:
Yes, tenantId has been deprecated for removal in favor of tenantID inside the SPC

Reply:
Ok, thanks. I see the PR here -- https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/1057 for that change. Checking with Anish if its a backcompat change.

Reply:
confirmed that this change is back compat (meaning Id and ID would work).